Compiler.cmake

```diff
@@ -157,14 +131,8 @@ elseif(CMAKE_COMPILER_IS_XLCXX)
   set(CMAKE_C_FLAGS_RELEASE ${COMMON_C_FLAGS_RELEASE})
```

Comment:
Can this be removed, and set, so CMAKE_C(XX)_FLAGS(_RELEASE|...) are set all in the same place?